src/services/ldap/hooks/index.js

```diff
@@ -18,7 +18,7 @@ exports.before = {
 	],
 	create: [disallow()],
 	update: [disallow()],
-	patch: [disallow()],
+	patch: [authenticate('jwt'), hasPermission('SCHOOL_EDIT')],
```

Comment:
Codacy found an issue: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5718258)